.pre-commit-config.yaml

```diff
@@ -13,27 +15,27 @@ repos:
     rev: v1.8.0
     hooks:
       # See https://github.com/pre-commit/mirrors-mypy/blob/main/.pre-commit-hooks.yaml
-     - id: mypy
-       types_or: [ python, pyi ]
-       args: [--ignore-missing-imports, --scripts-are-modules]
+      - id: mypy
+        types_or: [python, pyi]
+        args: [--ignore-missing-imports, --scripts-are-modules]
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.2.0
+    rev: v0.2.2
     hooks:
       - id: ruff
-        args: [ --fix, --exit-non-zero-on-fix ]
+        args: [--fix, --exit-non-zero-on-fix]
       - id: ruff-format
   - repo: https://github.com/PyCQA/bandit
     rev: 1.7.7
     hooks:
-    - id: bandit
-      args: [-c, pyproject.toml]
-      additional_dependencies: ["bandit[toml]"]
+      - id: bandit
+        args: [-c, pyproject.toml]
+        additional_dependencies: ["bandit[toml]"]
```

Comment:
I agree with these indents here. But Nicho noted them on another PR from me. Do we have a projet standard on these idents/formatting things?

Reply:
The rest of the file uses this style of indentation. From a quick look we also use it in the other files.

AFAIK, we do not have a standard defined.

Reply:
We don't, but at least each file should be consistent. FWIW, I consider whatever defaults the YAML plugin in VS Code has my standard (which is why the file got reformatted when I saved it).